stripe/src/main/java/com/stripe/android/PaymentSessionConfig.kt

```diff
@@ -37,7 +38,8 @@ data class PaymentSessionConfig internal constructor(
         private var addPaymentMethodFooter: Int = 0
 
         /**
-         * @param hiddenShippingInfoFields fields that should be hidden in the [ShippingInfoWidget]
+         * @param hiddenShippingInfoFields [CustomizableShippingField] fields that should be
```

Comment:
What's CustomizableShippingField?  

Could we just ~copy and paste this:
> Sets which address fields should be hidden in the shipping information screen. Values must be from among the labels listed in ShippingInfoWidget.. All fields will be shown if this list is empty. Note that not all fields can be hidden, such as country or name.

Reply:
`CustomizableShippingField` is an enum that maps to shipping fields, so it's important that the user is aware of this enum.

Reply:
Ah that's the actual values that you set here, that's perfect.